src/doc.d

```diff
@@ -379,6 +379,7 @@ RPAREN= )
 BACKTICK= `
 DOLLAR= $
 DEPRECATED= $0
+NOTHING =
```

Comment:
This was added as a workaround to the macros being trimmed. In order to maintain BC there needs to be a space before the `if (...)`, but in order to have a literal space there needs to be some other token first. In this case by defining a macro that prints nothing we can put a space after it and it won't be trimmed (e.g. `$(NOTHING) <- space` like on line 454)